apps/api/src/app/events/e2e/trigger-event-topic.e2e.ts

```diff
@@ -5,11 +5,12 @@ import {
   NotificationTemplateEntity,
   SubscriberEntity,
 } from '@novu/dal';
-import { ISubscribersDefine, ITopic, TriggerRecipients } from '@novu/node';
```

Comment:
Didn't make sense to me an interface that is shared in the monorepo to be exported from the Node.js client package.